consent/types.go

```diff
@@ -96,7 +96,7 @@ type HandledConsentRequest struct {
 	// authorization will be remembered indefinitely.
 	RememberFor int `json:"remember_for"`
 
-	ConsentRequest  *ConsentRequest     `json:"-"`
+	ConsentRequest  *ConsentRequest     `json:"consent_request"`
```

Comment:
The problem with exposing this is that acceptance of the consent request could overwrite this value and potentially mess stuff up. I haven't checked if that's actually the case but it could be an issue in some regression. I think it would be ok to copy this struct 1:1 and there set the `consent_request` value. If the order of the fields is the same, you can easily convert the type:

```go
var foo HandledConsentRequest
var bar = NewTypeWithJSON(HandledConsentRequest)
```

Reply:
I called it `HandledConsentRequestResponse` now but other suggestions are very welcome

Reply:
How about `PreviousConsentSession`? :)